python/Taskfile.yml

```
@@ -0,0 +1,63 @@
+# https://taskfile.dev
+# yaml-language-server: $schema=https://taskfile.dev/schema.json
+version: "3"
+
+includes:
+  ommx:
+    taskfile: ommx/Taskfile.yml
+    dir: ommx
+  ommx-openjij-adapter:
+    taskfile: ommx-openjij-adapter/Taskfile.yml
+    dir: ommx-openjij-adapter
+  ommx-python-mip-adapter:
+    taskfile: ommx-python-mip-adapter/Taskfile.yml
+    dir: ommx-python-mip-adapter
+  ommx-pyscipopt-adapter:
+    taskfile: ommx-pyscipopt-adapter/Taskfile.yml
+    dir: ommx-pyscipopt-adapter
+
+  # For python:doc task
+  api_reference:
+    taskfile: ../docs/api_reference/Taskfile.yml
+    dir: ../docs/api_reference
+    internal: true
+
+tasks:
+  default:
+    desc: Sync and Run tests for all Python projects
+    cmds:
+      - task: sync
+      - task: test
+
+  doc:
+    desc: Build and Open API reference documentation
+    cmds:
+      - task: api_reference
+
+  sync:
+    desc: Setup Python development environment, Rebuild Python SDK
+    cmds:
+      - uv sync --all-extras
+
+  format:
+    desc: Format Python code
+    cmds:
+      - uv run ruff format
+
+  test:
+    desc: Run tests for all Python projects
+    cmds:
+      - task: ommx:test
+      - task: ommx-openjij-adapter:test
+      - task: ommx-python-mip-adapter:test
+      - task: ommx-pyscipopt-adapter:test
+
+  set-version:
+    desc: Set the version for all Python projects
+    cmds:
+      - uv run set-version.py {{.ARGS}}
```

[According to the document](https://taskfile.dev/usage/#forwarding-cli-arguments-to-commands), arguments after `--` is passed as `CLI_ARGS`, not `ARGS`. And indeed, if we call `task python:test-version - 9.9.9` with the current implementation, it bumps version to `1.8.5`, instead of `9.9.9`.

After the following change, `task python:set-version -- 9.9.9` works as expected:

```suggestion
      - uv run set-version.py {{.CLI_ARGS}}
```

---

Fixed in https://github.com/Jij-Inc/ommx/pull/327/commits/d437bbb679f9db038a99d166e6589dead52c7f96